CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+# 0.1.0
+- [feature] account support
```

Comment:
that excluding domains is something new?
and that `prospect_filter_titles` setting? Maybe let's make the changelog include that details too.